template/pagetype-doc.html.jinja

```diff
@@ -29,6 +29,60 @@
     <div class="content">
         {{ content }}
     </div>
+    <div class="popup">
+      <div class="popup-window" id="myPopup">
+        <button type="button" class="close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        <p class="popup-text">Have questions?</p>
+        <a href="http://xrpldevs.com" class="btn btn-primary btn-arrow">
+          <span class="discord-logo"></span>
+          <span>Join the XRPL Discord</span>
+        </a>
+        <label class="show-again">
+        <input type="checkbox" class="show-again-checkbox">
+        <span class="checkmark"></span>
+          <label for="dont-show-again" class="show-again-message">Don't show this again</label>
+        </label>
+      </div>
+    </div>
+
+    <script type="text/javascript">
+      function localStorageAvailable() {
```

Comment:
This should be using `localstorage.getItem` and `localStorage.setItem` instead of directly setting and accessing this property.